### ISSUE TITLE
Question: why iOS platform not supported yet!

### ISSUE THREAD

**Issue by kareemradwan** [PapaTutuWawa/moxxyv2#14]
what is the technical limitation about support iOS Platform ?

thanks all 

@PapaTutuWawa

**Comment by ItsRogueRen** [comment]
> what is the technical limitation about support iOS Platform ?
> 
> 
> thanks all 
> 
> @PapaTutuWawa 

iOS is a notorious pain to develop FOSS software for. iOS apps have to be developed on a Mac computer, side-loading might as well be nonexistent on iPhones, the app store doesn't allow some FOSS licenses so they are required to dual license, and putting applications on the app store costs money


I can't say for certain this is why, but those are the most common reasons small FOSS projects tend to skip over iOS

**Comment by PapaTutuWawa** [comment]
@kareemradwan The most problem is that I cannot develop and/or test for iOS. In order to fully support the XMPP core RFC and XEP-0368 for direct TLS connections I need to be able to query DNS SRV records. Dart does not support it, Flutter does not support it, so I had to write platform native code. Moreover, I have the need for some extra platform-specific code that I cannot develop for iOS due to me not having access to the development and testing tools.

However, the reason I started with React Native and then went to Flutter is that it is possible to run the app on iOS. I would love to support iOS (and maybe others in the future) but I simply cannot.
